models/src_mailchimp.yml

```diff
@@ -2,13 +2,572 @@ version: 2
+
+          - name: delay_amount
+            description: The delay amount for an Automation email.
+
+          - name: delay_direction
+            description: Whether the delay settings describe before or after the delay action of an Automation email.
+
+          - name: delay_full_description
+            description: The user-friendly description of the delay and trigger action settings for an Automation email.
+
+          - name: delay_type
+            description: The type of delay for an Automation email.
+
+          - name: drag_and_drop
+            description: Whether the automation email uses the drag-and-drop editor.
+
+          - name: fb_comments
+            description: Allows Facebook comments on the automation email (also force-enables the Campaign Archive toolbar). Defaults to true.
+
+          - name: folder_id
+            description: If the automation email is listed in a folder, the ID for that folder.
+
+          - name: google_analytics
+            description: The custom slug for Google Analytics tracking (max of 50 bytes).
+
+          - name: inline_css
+            description: Automatically inline the CSS included with the automation email content.
+
+          - name: position
+            description: The position of an Automation email in a workflow.
+
+          - name: template_id
+            description: The ID for the template used in this automation email.
+
+          - name: timewarp
+            description: Send this automation email using Timewarpe_this
+
+          - name: track_ecomm_360
+            description: Whether to enable eCommerce360 tracking.
+
+          - name: track_goals
+            description: Whether to enable Goal tracking.
+
+          - name: track_html_clicks
+            description: Whether to track clicks in the HTML version of the automation email. Defaults to true. Cannot be set to false for variate campaigns.
+
+          - name: track_opens
+            description: Whether to track opens. Defaults to true. Cannot be set to false for variate automation emails.
+
+          - name: track_text_clicks
+            description: Whether to track clicks in the plain-text version of the automation email. Defaults to true. Cannot be set to false for variate campaigns.
+
+          - name: use_conversation
+            description: Use Mailchimp Conversation feature to manage out-of-office replies.
+            
+      - name: automation_recipient
+        description: Each record represents info about the recipient of each automation.
+        columns:
+          - name: member_id
+            description: The ID of the member the automation was sent to.
+
+          - name: automation_email_id
+            description: The ID for the automation email.
+
+          - name: list_id
+            description: The ID of the List an automation was sent to.
+
+      - name: automation_recipient_activity
```

Comment:
I think the field definition for `action` was missed in this as well as the stg yml.